factory/defaults/chains/mainnet.yaml

```diff
@@ -26,6 +26,15 @@ chains:
       coin_market_cap:
         chain_id:
         asset_id: "6756"
+  ADA:
+    chain: ADA
+    driver: cardano
+    chain_id: cardano
+    network: mainnet
+    chain_name: Cardano
+    decimals: 6
+    fee_limit: "3.0"
```

Comment:
I'd set FeeLimit higher, perhaps `30.0` ADA, which is only like $20.  Treasury will record this trust-on-first-use style, and use as a sanity limit for future transactions.